test/helpers.test.ts

```diff
@@ -1,9 +1,13 @@
-import { describe, expect, test } from 'vitest';
-import { parseNotarizationInfo } from '../src/helpers';
+import fs from 'node:fs';
+import os from 'node:os';
+import path from 'node:path';
+
+import { describe, expect, it } from 'vitest';
+import { parseNotarizationInfo, withTempDir } from '../src/helpers';
 
 describe('helpers', () => {
   describe('parseNotarizationInfo', () => {
-    test('build a NotarizationInfo object', () => {
+    it('build a NotarizationInfo object', () => {
```

Comment:
Nit: verb tense consistency

```suggestion
    it('builds a NotarizationInfo object', () => {
```